docs/server/settings.rst

```diff
@@ -501,7 +501,6 @@ Translation environment configuration settings.
   Current options:
 
   - Translate Toolkit (default) - translate.storage.statsdb.wordcount
-  - Pootle - pootle.core.utils.wordcount.wordcount
```

Comment:
I think it is a valid pattern to overwrite wordcount function. probs it can be implemented using getter.